src/main/java/commands/TodoCommand.java

```diff
@@ -0,0 +1,53 @@
+package commands;
+
+import java.io.IOException;
+
+import exception.TodoFormatException;
+import storage.Storage;
+import task.Task;
+import task.TaskList;
+import task.Todo;
+
+/**
+ * Represents the command used to add a todo to the task list.
+ */
+public class TodoCommand extends Command {
+
+    public static final String COMMAND_WORD = "todo";
```

Comment:
Any reason for adding this constant to each command class rather than using the string directly in Parser?